SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/TimeToLive.cs

```diff
@@ -21,8 +21,8 @@ public static class TimeToLive
         public static readonly TimeSpan NoCache = TimeSpan.Zero;
 
         /// <summary>
-        /// Default caching value of one day.
+        /// Default caching value is cache does not expire.
         /// </summary>
-        public static readonly TimeSpan Default = TimeSpan.FromDays(1);
+        public static readonly TimeSpan Default = Infinite;
```

Comment:
I don't think we map Infinite correctly back to Java and objective C in the conversion extensions.

Reply:
seems correct... 🤔

Reply:
ok its not the Infinite timespan from c# but our own